packages/sui-lint/eslintrc.js

```diff
@@ -69,9 +69,7 @@ module.exports = {
     'standard',
     'standard-react',
     'plugin:cypress/recommended',
-    'prettier',
-    'prettier/standard',
-    'prettier/react'
+    'prettier'
```

Comment:
Now prettier should be extended only once. Sub-packages are already in the general one.